public/js/selectorFuction.js

```diff
@@ -1,15 +1,10 @@
+// eslint-disable-next-line no-unused-vars
 const querySelectors = (selectorsName, enterTypeofQuery) => {
   if (selectorsName.length !== enterTypeofQuery.length) return false;
   const elements = {};
-  enterTypeofQuery.map(
-    (e, i) => {
-      elements[selectorsName[i]] = document.querySelector(e);
-      return '';
-    },
-  );
+  enterTypeofQuery.map((e, i) => {
+    elements[selectorsName[i]] = document.querySelector(e);
+    return '';
+  });
```

Comment:
@aminalakhsham I just noticed that you aren't using the return value of this map, so you could change it to a `forEach` and then you could remove the `return '';` 🤷‍♂️